openedx/core/djangoapps/catalog/tests/test_utils.py

```diff
@@ -152,6 +154,56 @@ def test_get_one(self, mock_warning, _mock_info):
         self.assertEqual(actual_program, expected_program)
         self.assertFalse(mock_warning.called)
 
+    @mock.patch(UTILS_MODULE + '.get_programs')
+    @mock.patch(UTILS_MODULE + '.get_program_types')
+    def test_get_programs_with_type(self, mock_get_program_types, mock_get_programs, _mock_warning, _mock_info):
```

Comment:
@ibrahimahmed443 whoops, that's true. Please put these tests in a new class instead of jamming them into the existing class. The idea in this test file is to give each function its own test class, for clarity.